src/common/sound.cpp

```diff
@@ -14,6 +14,34 @@ Sound::Sound() {
     frequency = 100.f; // frequency of sound signal (0-1000)
     volume = 0.50;     // volume of sound signal (0-1)
 
+    // -- values of sound signals - frequencies, volumes, durations
+    // -- durations of beep
+    durations[eSOUND_TYPE_ButtonEcho] = 100.f;
+    durations[eSOUND_TYPE_StandardPrompt] = 500.f;
+    durations[eSOUND_TYPE_StandardAlert] = 200.f;
+    durations[eSOUND_TYPE_CriticalAlert] = 500.f;
+    durations[eSOUND_TYPE_EncoderMove] = 50.f;
+    durations[eSOUND_TYPE_BlindAlert] = 100.f;
+    durations[eSOUND_TYPE_Start] = 100.f;
+
+    // -- frequencies of beep
+    frequencies[eSOUND_TYPE_ButtonEcho] = 900.f;
+    frequencies[eSOUND_TYPE_StandardPrompt] = 600.f;
```

Comment:
frequencies are double? Why? Double is not HW accelerated on our CPU? Assigning a float into a double?

Reply:
Another mistake of not checking my types. Fixed.